src/radio-button/index.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+'use client';
+import React from 'react';
+import clsx from 'clsx';
+
+import InternalRadioButton from '../internal/components/radio-button';
+import useBaseComponent from '../internal/hooks/use-base-component';
+import { applyDisplayName } from '../internal/utils/apply-display-name';
+import { RadioButtonProps } from './interfaces';
+
+import styles from './styles.css.js';
+
+export { RadioButtonProps };
+
+const RadioButton = React.forwardRef((props: RadioButtonProps, ref: React.Ref<HTMLInputElement>) => {
```

Comment:
should we use `{ disabled = false, readOnly = false, ...props }` here so that there props default value is correctly assigned with documenter?